R/adults.R

```diff
@@ -1,85 +1,66 @@
-#' @param prop_delta_trans
-#' @param prop_bay_trans
-#' @param betas Parameters estimated from calibration
-#' @section Parameters:
-#' Parameters from the model are obtained from either literature, calibration, export elicitation,
-#' or meta-analysis. The source for each parameter in this function are detailed below.
-#'
-#'
-#' \itemize{
-#' \item intercept- Intercept: Empirical model fit using 2008–2011 tagging data provided by East Bay Municipal Utility District
-#' \item wild - Wild Salmon Parameter: Estimated with coded wire tag data 2010–2013 \href{https://nrm.dfg.ca.gov/FileHandler.ashx?DocumentID=162355&usg= AOvVaw0VgMOwD7knFfSxRZy6k8RG}{(Kormos et al. 2012, Palmer-Zwahlen & Kormos 2013-2015, Palmer-Zwahlen et al. 2018)}
-#' \item natal flow - Natal Flow Parameter: Empirical model fit using 2008–2011 tagging data provided by East Bay Municipal Utility District
-#' \item gates open - Cross Channel Gates Parameter: Empirical model fit using  2008–2011 tagging data provided by East Bay Municipal Utility District.
-#' \item bay - Bay transport proportion parameter. Estimated with coded wire tag data 2010–2013 \href{https://nrm.dfg.ca.gov/FileHandler.ashx?DocumentID=162355&usg= AOvVaw0VgMOwD7knFfSxRZy6k8RG}{(Kormos et al. 2012, Palmer-Zwahlen & Kormos 2013-2015, Palmer-Zwahlen et al. 2018)}
-#' \item delta  - Delta transport proportion parameter. Estimated with coded wire tag data 2010–2013 \href{https://nrm.dfg.ca.gov/FileHandler.ashx?DocumentID=162355&usg= AOvVaw0VgMOwD7knFfSxRZy6k8RG}{(Kormos et al. 2012, Palmer-Zwahlen & Kormos 2013-2015, Palmer-Zwahlen et al. 2018)}
-#' }
+#' @param wild Variable indicator of wild fish returning
+#' @param natal_flow Variable describing proportion flows at tributary junctions coming from natal watershed in October
+#' @param south_delta_watershed Variable indicator if watershed feeds into South Delta
+#' @param cross_channel_gates_closed Variable describing number of days gates are closed for each month
+#' @param prop_bay_trans Variable describing proportion transport to the bay
+#' @param prop_delta_trans Variable describing proportion transport to the delta
+#' @param .intercept Intercept, source: Empirical model fit using 2008–2011 tagging data provided by East Bay Municipal Utility District
+#' @param .wild Coefficient for wild variable, source: Estimated with coded wire tag data 2010–2013 \href{https://nrm.dfg.ca.gov/FileHandler.ashx?DocumentID=162355&usg= AOvVaw0VgMOwD7knFfSxRZy6k8RG}{(Kormos et al. 2012, Palmer-Zwahlen & Kormos 2013-2015, Palmer-Zwahlen et al. 2018)}
+#' @param .natal_flow Coefficient for natal flow variable, source: Empirical model fit using 2008–2011 tagging data provided by East Bay Municipal Utility District
+#' @param .cross_channel_gates_closed Coefficient for cross_channel_gates_closed variable, Source: Empirical model fit using  2008–2011 tagging data provided by East Bay Municipal Utility District.
```

Comment:
for these empirical model fit params, it would be good to have Rmarkdown of those analyses... a TODO for the future